src/clojure/nrepl/server.clj

```diff
@@ -82,27 +83,41 @@
 
 (def default-middlewares
   "Middleware vars that are implicitly merged with any additional
-   middlewares provided to nrepl.server/default-handler."
+   middleware provided to nrepl.server/default-handler.
 
+   Note: this is part of the de-facto API, thus the name of the var should be
```

Comment:
I'd just add one more `def` with the legacy name and mention it exists only for backwards compatibility.

Reply:
done